Documentation/ApiOverview/SiteHandling/UseSiteInTypoScript.rst

```diff
@@ -80,3 +80,17 @@ In the Fluid template the properties of the site entity can be accessed with:
 
    <p>{site.rootPageId}</p>
    <p>{site.configuration.someCustomConfiguration}</p>
```

Comment:
actually we could remove this line then

Reply:
Actually, the "configuration"
Key hete references contents of "config.yaml" outside the settings scope - that would still apply IMO?!

Reply:
Yup, not everything are settings, but configuration.